iommu_registers.adoc

```diff
@@ -249,6 +249,18 @@ operations. Support for device initiated atomic memory operations must be
 discovered through other means.
 ====
 
+[NOTE]
+====
+The IOMMU is designed to provide a highly modular and extensible set of
+capabilities allowing implementations to include only the exact set of
+capabilities required for an application. In addition, users may add
+their own custom extensions to the IOMMU.
+
+RISC-V platform and profile specifications may mandate a set of IOMMU
```

Comment:
The IOMMU must support all the virtual memory extensions that are supported by any of the harts in the implementation.